internal/pkg/describe/app.go

```diff
@@ -46,6 +47,7 @@ func (a *App) HumanString() string {
 	fmt.Fprint(writer, color.Bold.Sprint("About\n\n"))
 	writer.Flush()
 	fmt.Fprintf(writer, "  %s\t%s\n", "Name", a.Name)
+	fmt.Fprintf(writer, "  %s\t%s (latest: %s)\n", "Version", a.Version, deploy.LatestAppTemplateVersion)
```

Comment:
what do you think of coloring if `a.Version != deploy.LatestAppTemplateVersion`?

```go
availableVersion := "\n"
if deploy.LatestAppTemplateVersion != a.Version {
    availableVersion = color.Yellow.Sprintf("(latest available: %s)\n", deploy.LatestAppTemplateVersion)
}
```

Reply:
oh that's a good idea